addons/website_sale/views/generate_primary_template.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<function model="ir.module.module" name="_generate_primary_snippet_templates">
+    <value eval="[ref('base.module_website_sale')]"/>
+</function>
```

Comment:
What does this do exactly?
Seems like something that should only be used in a theme :thinking: 

Edit: You told me why, but it should be clearly explained here in a comment.

Reply:
When I said 'here' I meant in the code with a more generic explanation in the xml directly before the `function` call, but thanks for the explanation :+1: (will still be useful if people land on this PR looking for more information about your changes).